orchestrator/Cargo.toml

```diff
@@ -41,3 +41,15 @@ gorc = { path = "./gorc" }
 relayer = { path = "./relayer" }
 gravity_abi_build = { path = "./gravity_abi_build" }
 gravity_abi = { path = "./gravity_abi" }
+
+[features]
+ethermint = [
+    "orchestrator/ethermint",
+    "cosmos_gravity/ethermint",
+    "relayer/ethermint",
+    "gorc/ethermint",
+    "register_delegate_keys/ethermint",
+]
+
+[patch.crates-io]
+deep_space = { git = "https://github.com/crypto-org-chain/deep_space/", branch = "5fb0457-cronos" }
```

Comment:
Changed can be viewed there:

https://github.com/crypto-org-chain/deep_space/commit/01008a1859382331999729ebeae3245b0cd61ae5